build-packages/check-pr/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "check-pr",
+  "version": "2.12.0",
+  "description": "Checks the current PR for semantical correctness and appropriate changelog.",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "compile": "tsc -p tsconfig.json",
+    "lint": "eslint --ignore-pattern '!index.ts' && prettier --check '**/*.ts'",
+    "lint:fix": "eslint --ignore-pattern '!index.ts' --fix --quiet && prettier --write '**/*.ts'",
+    "check:dependencies": "depcheck --skip-missing=true .",
+    "package": "ncc build lib/index.js --out ../../.github/actions/check-pr/",
+    "all": "yarn && yarn run compile && yarn run lint && yarn run package"
```

```suggestion
    "compile": "tsc -p tsconfig.json",
    "lint": "eslint --ignore-pattern '!index.ts' && prettier --check '**/*.ts'",
    "lint:fix": "eslint --ignore-pattern '!index.ts' --fix --quiet && prettier --write '**/*.ts'",
    "check:dependencies": "depcheck --skip-missing=true .",
    "postcompile": "ncc build lib/index.js --out ../../.github/actions/check-pr/",
    "all": "yarn && yarn run lint && yarn run compile"
```
[nth] `lint` can and should run before `compile`
[pp] If `package` is a follow up of `compile` (either through renaming to `postcompile` or by adding it to `turbo.json` accordingly), then we build and package regularly... would be nice 😊